.github/workflows/subset_report.yml

```diff
@@ -0,0 +1,81 @@
+name: 'Subset report'
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'src/ontology/uberon-edit.obo'
+  workflow_dispatch:
+env:
+  STATUS: 1
+jobs:
+  is_changed:
+    runs-on: macos-latest
+    # container: obolibrary/odkfull:latest
+    steps:
+      - uses: docker-practice/actions-setup-docker@master
+      - run: |
+          set -x
+          docker version
+      - name: Install ODK
+        run: docker pull obolibrary/odklite
+      # Checks-out current branch
+      - uses: actions/checkout@v2
+      # Checks-out main and prepare git diff report
+      - name: create git diff
+        id: gitdiff
+        if: ${{ github.event.pull_request.base.sha }}
+        run: |
+          git fetch origin main ${{ github.event.pull_request.base.sha }}
+          ls -l
+          git diff ${{ github.event.pull_request.base.sha }} ${{ github.sha }} src/ontology/uberon-edit.obo > gitdiff.md
+          cat gitdiff.md
+          echo ::set-output name=diff::$(cat gitdiff.md)
+      # Set Status to 0 to cancel the workflow' if keyword not in the diff file
+      - name: conditional cancel
+        if: "!contains(steps.gitdiff.outputs.diff, 'never_in_taxon')"
```

Comment:
I can test it if I can have good `uberon-edit.obo` samples

Reply:
This will only work for `never_in_taxon`. @ubyndr can you make sure this works for any of these:

in_taxon
never_in_taxon
only_in_taxon
present_in_taxon